motion/p5js/motion-triangle/index.md

```diff
@@ -0,0 +1,9 @@
+---  
+layout: p5js
+author: Daniele Maselli
+author-url: https://github.com/DanieleMaselli
+title: motion-triangle
+tags: [P5.js, p5.Vector, color, triangle]
+permalink: /color/motion/
+summary: primary colors represented by a moving triangle on P5.js.
```

Comment:
what wrong about that? sorry not my language ; ) 
primary color expressed by a moving triangle with P5.js? primary color illustrated by a moving triangle with P5.js?